packages/components/psammead-brand/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 3.0.0   | [PR#528](https://github.com/bbc/psammead/pull/528) Add link prop to allow dynamic link |
```

Comment:
Should this be alpha?

Reply:
I don't think so. Why would it be alpha?

Reply:
It hasnt been swarmed since Phils refactor

Reply:
A swam was carried out on https://github.com/bbc/psammead/pull/480#issuecomment-493459076

Reply:
Ahh, curse me being on leave :D haha thanks